.luacheckrc

```diff
@@ -12,16 +12,17 @@ read_globals = {
     "Wait",
     "addHotkey",
     "broadcastToAll",
+    "broadcastToColor",
```

Comment:
this also should probably be removed from the list since i've run into scripting errors with this function before `Player[<color>].broadcast` is a safer way of doing that